posts/2022-09-01-22.0.0.10-beta.adoc

```diff
@@ -0,0 +1,581 @@
+    </library>
+
+    <!-- 
+        Configure the JCache cache manager.
+     -->
+    <cacheManager id="CacheManager" uri="uri://someuri">
+        <properties prop1="value1" prop2="value2" />
+
+        <cachingProvider libraryRef="JCacheProviderLib" />
+    </cacheManager>
+
+    <!--
+        Configure the HTTP session cache.
+     -->
+    <httpSessionCache cacheManagerRef="CacheManager" ... />
+```
+
+```
+========================================================================
+Configuring Multiple Caches
+========================================================================
+```
+> When configuring multiple distributed caches, instead of nesting the cacheManager configuration element within the cache element, the cache element needs to refer to the cache manager via the cacheRef attribute.
+
+```
+    <featureManager>
+        <feature>appSecurity-3.0</feature>
+        <feature>distributedSecurityCache-1.0</feature>
+        <feature>sessionCache-1.0</feature>
+    </featureManager>
+
+    <!-- 
+        The 3rd-party JCache provider library that Liberty will use to manage and connect to the cache.
+     -->
+    <library id="JCacheProviderLib">
+        <fileset dir="${shared.resource.dir}" includes="jcacheprovider.jar" />
+    </library>
+
+    <!-- 
+        Configure the JCache cache manager.
+     -->
+    <cacheManager id="CacheManager" uri="uri://someuri">
+        <properties prop1="value1" prop2="value2" />
+
+        <cachingProvider libraryRef="JCacheProviderLib" />
+    </cacheManager>
+
+    <!-- 
+        Configure the JCache cache instances.
+     -->
+    <cache id="AuthCache" name="AuthCache" cacheManagerRef="CacheManager" />
+    <cache id="LoggedOutCookieCache" name="LoggedOutCookieCache" cacheManagerRef="CacheManager" />
+
+    <!-- 
+        Configured the authentication cache, logged-out cookie cache and HTTP session cache.
+     -->
+    <authCache cacheRef="AuthCache" />
+    <webAppSecurity loggedoutCookieCacheRef="LoggedOutCookieCache" ... />
+    <httpSessionCache cacheManagerRef="CacheManager" ... />
+```
+   
+   - Where can they find out more about this specific update (eg Open Liberty docs, Javadoc) and/or the wider technology?
+
+- https://www.ibm.com/docs/en/was-liberty/base?topic=liberty-configuring-authentication-cache-in
+- https://www.ibm.com/docs/en/was-liberty/base?topic=manually-configuring-liberty-session-persistence-jcache
+
+
+
+
+To enable the new beta features in your app, add them to your `server.xml`:
+
+[source, xml]
+----
+
+----
+
+[#run]
+=== Try it now 
+
+To try out these features, just update your build tools to pull the Open Liberty All Beta Features package instead of the main release. The beta works with Java SE XX, Java SE 17, Java SE 11, and Java SE 8.
+// // // // // // // //
+// In the preceding section:
+// Check if a new non-LTS Java SE version is supported that needs to be added to the list (17, 11, and 8 are LTS and will remain for a while)
+// // // // // // // //

```

Comment:
```suggestion
To try out these features, just update your build tools to pull the Open Liberty All Beta Features package instead of the main release. The beta works with Java SE 19, Java SE 18, Java SE 17, Java SE 11, and Java SE 8, however, the Jakarta EE 10 features are only compatible with Java SE 11 and later.
```